app.rb

```diff
@@ -1,6 +1,7 @@
 # enoding: utf-8
 
 require 'sinatra'
+require 'fluent/version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.